portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,12 +33,26 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String commentcount = request.getParameter("count");
+    int maxcomments;
```

Comment:
camelcase for variable names.  Look at java style guide for variable names.

Reply:
Fixed